app/database/operator/server_data_operator/handlers/thread.ts

```diff
@@ -79,11 +79,13 @@ const ThreadHandler = (superclass: any) => class extends superclass {
         const threadParticipants = (await this.handleThreadParticipants({threadsParticipants, prepareRecordsOnly: true})) as ThreadParticipantModel[];
         batch.push(...threadParticipants);
 
-        const threadsInTeam = await this.handleThreadInTeam({
-            threadsMap: {[teamId]: threads},
-            prepareRecordsOnly: true,
-        }) as ThreadInTeamModel[];
-        batch.push(...threadsInTeam);
+        if (teamId) {
+            const threadsInTeam = await this.handleThreadInTeam({
```

Comment:
make sure that #6001  is setting `loadedInGlobalThreads` here I guess.

Reply:
But it doesn't exist in the model in this PR. Depending on what PR gets merged first, the other one can resolve it as a merge conflict.